### ISSUE TITLE
new gclone cannot find the folder when using the folder ID.

### ISSUE THREAD

**Issue by ray1025** [dogbutcat/gclone#1]
new gclone cannot find the folder when using the folder ID.

**Comment by dogbutcat** [comment]
emmm…… Cloud you give me an example? I'm only using drive copy, for me it working fine.

**Comment by dogbutcat** [comment]
I've just tried, it seems works. I don't know if your point is it.
```shell
> gclone lsf drive: --drive-root-folder-id=1n5wT3E0h3aH9bbe0vjf3m7HwVw3WuIM
白い巨塔.1080p.HDTV.x265.10bit.AC3￡cXcY@FRDS.nfo
白い巨塔.E01.2003.1080p.HDTV.x265.10bit.AC3￡cXcY@FRDS.mkv
白い巨塔.E02.2003.1080p.HDTV.x265.10bit.AC3￡cXcY@FRDS.mkv
白い巨塔.E03.2003.1080p.HDTV.x265.10bit.AC3￡cXcY@FRDS.mkv
白い巨塔.E04.2003.1080p.HDTV.x265.10bit.AC3￡cXcY@FRDS.mkv
白い巨塔.E05.2003.1080p.HDTV.x265.10bit.AC3￡cXcY@FRDS.mkv
白い巨塔.E06.2003.1080p.HDTV.x265.10bit.AC3￡cXcY@FRDS.mkv
白い巨塔.E07.2003.1080p.HDTV.x265.10bit.AC3￡cXcY@FRDS.mkv
白い巨塔.E08.2003.1080p.HDTV.x265.10bit.AC3￡cXcY@FRDS.mkv
白い巨塔.E09.2003.1080p.HDTV.x265.10bit.AC3￡cXcY@FRDS.mkv
白い巨塔.E10.2003.1080p.HDTV.x265.10bit.AC3￡cXcY@FRDS.mkv
白い巨塔.E11.2003.1080p.HDTV.x265.10bit.AC3￡cXcY@FRDS.mkv
白い巨塔.E12.2003.1080p.HDTV.x265.10bit.AC3￡cXcY@FRDS.mkv
白い巨塔.E13.2003.1080p.HDTV.x265.10bit.AC3￡cXcY@FRDS.mkv
白い巨塔.E14.2003.1080p.HDTV.x265.10bit.AC3￡cXcY@FRDS.mkv
白い巨塔.E15.2003.1080p.HDTV.x265.10bit.AC3￡cXcY@FRDS.mkv
白い巨塔.E16.2003.1080p.HDTV.x265.10bit.AC3￡cXcY@FRDS.mkv
白い巨塔.E17.2003.1080p.HDTV.x265.10bit.AC3￡cXcY@FRDS.mkv
白い巨塔.E18.2003.1080p.HDTV.x265.10bit.AC3￡cXcY@FRDS.mkv
白い巨塔.E19.2003.1080p.HDTV.x265.10bit.AC3￡cXcY@FRDS.mkv
白い巨塔.E20.2003.1080p.HDTV.x265.10bit.AC3￡cXcY@FRDS.mkv
白い巨塔.E21.2003.1080p.HDTV.x265.10bit.AC3￡cXcY@FRDS.mkv
```

**Comment by ray1025** [comment]
gclone copy gc:{1evOG_BeuyjkpRvajfWhbqyrvlOmxqR4q} gc:{1oDCEAgXzHfXjMGbZfmQu65288jhw995-} --drive-server-side-across-configs

**Comment by ray1025** [comment]
Failed to copy: directory not found

**Comment by ray1025** [comment]
However, it(https://github.com/donwa/gclone) is normal for me to use this.

**Comment by dogbutcat** [comment]
OK, I've got it. I just checked original rclone `rclone-v1.51.0-osx-amd64` with command `lsf`, also pump to me these error:
```
ERROR : : error listing: directory not found
Failed to lsf with 2 errors: last error was: error in ListJSON: directory not found
```
I'll post here after dig it further.

**Comment by ray1025** [comment]
Many thanks.

**Comment by dogbutcat** [comment]
Hi!
Your problem has FIXED in version [v1.56.1-mod1.3.2](https://github.com/dogbutcat/gclone/releases/tag/v1.56.1-mod1.3.2)
Feel free to download and have a try!

**Comment by ray1025** [comment]
everything is normal.
